### PR TITLE
Multiple hosts: unify menu style across hosts

### PR DESCRIPTION
**Pull request by jakubjezek001**
## Brief description
Some hosts were missing Library loader menu item

## Description
This will allow to use Library loader across most of supported DCC

## Additional info
Triple dot stile `Library...` was unified to keep one style only without space between word

## Testing notes:
1. open your host
2. see there is available `Library...` under `Manage...`
3. It should open Library Loader
4. Also check the triple dots are not having space between word

# Tested hosts
- [x] Houdini
- [x] Nuke
- [x] Hiero
- [x] Fusion
- [x] Resolve
- [x] Flame

### CODE REVIEW

File: openpype/hosts/houdini/startup/MainMenuCommon.xml [hunk 1]
```diff
@@ -14,18 +14,18 @@ host_tools.show_creator(parent)
 ]]></scriptCode>
             </scriptItem>
 
-            <scriptItem id="avalon_load">
```
Comment (iLLiCiTiT) [inline openpype/hosts/houdini/startup/MainMenuCommon.xml:17]: You have removed loader...

Comment (jakubjezek001) [reply openpype/hosts/houdini/startup/MainMenuCommon.xml:17]: huh.. perhaps I felt it was not useful anymore. Joking, thanks - nice catch! ;)

**Review by iLLiCiTiT — APPROVED** [review]
Maya , Nuke and Houdini.

**Review by BigRoy — COMMENTED** [review]
**Note:** Flame and Unreal still seem to have a space between the words and the triple dots.

Should that get patched in this PR too? The PR title doesn't mention it but the description does say "unify style across hosts".

**Comment by jakubjezek001** [comment]
> unify style across hosts

good catch, I will change the topic and fix the rest. Thaks for noticig ;)

**Comment by mkolar** [comment]
Task linked: [OP-2629 hosts: menu style unification](https://app.clickup.com/t/6658547/OP-2629)

**Comment by jakubjezek001** [comment]
the Unreal integration will be soon merged into OpenPype codebase from spreaded repositories and the menu style unification will be part of the move.. https://github.com/pypeclub/OpenPype/issues/2433

**Review by m-u-r-p-h-y — APPROVED** [review]
blender is loading ok

image

**Pull request merged**